b2/sync/policy.py

```diff
@@ -246,8 +245,7 @@ def make_b2_delete_actions(source_file, dest_file, dest_folder, transferred):
         keep = (version_index == 0) and (source_file is not None) and not transferred
         if not keep:
             yield B2DeleteAction(
-                dest_file.name,
-                dest_folder.make_full_path(dest_file.name), version.id_,
+                dest_file.name, dest_folder.make_full_path(dest_file.name), version.id_,
                 make_b2_delete_note(version, version_index, transferred)
```

Comment:
this gets poorly formatted due to missing , at the end of last value, please correct it